apps/editing-toolkit/editing-toolkit-plugin/maps-provider/index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Map provider
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Adds a global variable containing the map provider to the window object
+ *
+ * @return void
+ */
+function set_map_provider() {
+	$map_provider = apply_filters( 'wpcom_map_block_map_provider', 'mapbox' );
+
+	wp_register_script( 'dummy-script', '', array(), '1.0', false );
+	wp_localize_script( 'dummy-script', 'map_block_map_provider', $map_provider );
+	wp_enqueue_script( 'dummy-script', '', array(), '1.0', false );
```

Comment:
I think we can give it a specific handle name like `map-block-map-provider` just to make debugging easier (if someone were to var_dump the list of script handles internally).

And is calling `wp_register_script` and `wp_enqueue_script` necessary for `wp_localize_script` to work or can they be omitted?

Reply:
I changed the script name. 

> And is calling wp_register_script and wp_enqueue_script necessary for wp_localize_script to work or can they be omitted?

I believe they are.